doc/lua_api.txt

```diff
@@ -971,20 +971,64 @@ There are a bunch of different looking node types.
 Look for examples in `games/minimal` or `games/minetest_game`.
 
 * `normal`
+    * A node-sized cube.
 * `airlike`
+    * Invisible, uses no texture.
 * `liquid`
+    * The cubic source node for a liquid.
 * `flowingliquid`
+    * The flowing version of a liquid, appearing with various heights and
+      slopes.
 * `glasslike`
+    * Often used for partially-transparent nodes.
+    * Only external sides of textures are visible.
 * `glasslike_framed`
+    * All face-connected nodes are drawn as one volume within a surrounding
+      frame.
+    * The frame appearence is generated from the edge pixels of the first
+      texture specified in `tiles` (preferably 16x16 pixels).
+    * The glass 'shine' (or other desired detail) on each node face is supplied
+      by the second texture specified in `tiles`.
 * `glasslike_framed_optional`
+    * This switches between the above 2 drawtypes according to the clientside
```

Comment:
For consistency, note both "client side" and "client-side" are already used in the document, but that this would be the first use of "clientside". I think we should avoid a third version. :)

Reply:
Ok.